### DISCUSSION TITLE
Normalization of Shannon-Wiener diversity index and absolute number of clonotypes?

### DISCUSSION THREAD

**Discussion by JoostKoedijk** [milaboratory/mixcr#1785] [Q&A] [answered]
Hi everyone, 

I am new to TCR analysis and using MIXCR. I have applied MIXCR to bulk RNA-sequencing data to identify the extent of T cell clonality in cancer patients. Towards this end, I am using the CDR3 spectratyping and diversity results for the TCR beta subunit. In papers, I sometimes noticed a normalization for the absolute numbers of clones. 

Should I normalize the absolute number of clones for the total sequencing depth? (/# of reads). Or does MIXCR normalize this already? I guess this normalization does not apply to the Shannon-Wiener diversity index, or does it? 

Many thanks in advance for your time!

**Comment by mizraelson** [comment, marked as answer]
Hi,
You usually need to normalize your data to compare diversity between samples. The best practice is to normalize by the number of UMIs, if available; otherwise, the number of reads should also work.

**Reply by JoostKoedijk** [reply]
Many thanks! Normally, this normalization is integrated with the default downsampling, correct?

**Reply by mizraelson** [reply]
Yes, when you specify the `--default-downsampling` parameter with `mixcr postanalysis`, it performs normalization prior to calculating the metrics.
